### PR TITLE
Check CA status: add HTTP timeout

### PR DESCRIPTION
**Pull request by MartinBasti**
https://pagure.io/freeipa/issue/6766

### CODE REVIEW

File: ipalib/constants.py [hunk 1]
```diff
@@ -171,6 +171,9 @@
     ('ca_install_port', None),
     ('ca_agent_install_port', None),
     ('ca_ee_install_port', None),
+    # How long http connection should wait before trying again [seconds].
+    # Do not mistake with "startup_timeout"
+    ('ca_status_http_timeout', 30),
```
Comment (stlaz) [inline ipalib/constants.py]: I don't like adding this to api.env, if you really have to, can't you make this more general?

Comment (MartinBasti) [inline ipalib/constants.py]: Could be `http_timeout`?

Comment (stlaz) [inline ipalib/constants.py]: That's better.

**Comment by MartinBasti** [comment]
master:

* 20f7689079328aeef42b62a359b303f531db5666 http_request: add timeout option
* 05984f171b0b41681254c95380a0598e4208a201 ca_status: add HTTP timeout 30 seconds


ipa-4-5:

* 48bb3cb69c000cea3f28bd5b44072d0fe9caa7a2 http_request: add timeout option
* 68ce9aa2addb6048333e723f771132f5da7dd38f ca_status: add HTTP timeout 30 seconds

**Pull request closed without merging**